test/simulation/xudtest/node.go

```diff
@@ -51,13 +51,15 @@ func (cfg nodeConfig) genArgs() []string {
 	var args []string
 
 	args = append(args, "--initdb=false")
+	args = append(args, "--nosanitychecks=true")
```

Comment:
Why reenable this flag? I think it's handy to test the sanity checks automatically, and if they are not passing it's likely a sign that there's a bug in the implementation.

Reply:
When I resolved conflicts with master, it looked to me like you added it. 
If that wasn't the case, i'll remove it.

Reply:
Ah ok, must have just been some mix up then.